lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -196,41 +203,44 @@ button, a {
     width: 100%;
 
     &:hover {
-      background-color: var(--nx-swatch-indigo-95);
+      background-color: var(--nx-color-interactive-background-hover);
+      color: var(--nx-color-text-dark);
     }
 
     &.selected {
-      background-color: var(--nx-swatch-blue-90);
+      background-color: var(--nx-color-interactive-background-selected);
       font-weight: 600;
     }
 
     &:focus {
-      border-color: var(--nx-color-interactive-border-focus);
-      box-shadow: var(--nx-box-shadow-focus);
-      outline: none;
+      @extend %collapsible-items-focus-style;
     }
   }
 }
 
 // sidebars don't typically have the white background and so warrant different highlight colors
+$-sidebar-interactive-background-hover: var(--nx-swatch-indigo-90);
 .nx-page-sidebar {
   button, a {
     &.nx-collapsible-items__child {
       &:hover {
-        background-color: var(--nx-swatch-indigo-90);
+        background-color: $-sidebar-interactive-background-hover;
       }
 
       &.selected {
-        background-color: var(--nx-swatch-blue-10);
-        color: var(--nx-swatch-white);
+        background-color: var(--nx-color-interactive-background-selected);
       }
     }
   }
 
   .nx-collapsible-items__trigger {
     &:hover:not([disabled]) {
-      background-color: var(--nx-swatch-indigo-90);
+      background-color: $-sidebar-interactive-background-hover;
     }
-  }
 
+    &:active:not([disabled]),
+    &:active:not([disabled]):focus {
+      background-color: var(--nx-swatch-indigo-95);;
```

Comment:
> Looks like there's a --nx-color-border-secondary variable defined for this color. But not sure if it would be semantically correct? 

Hmm yeah, I think generally that color might be intended to be connected semantically but the variable feels like it's tied to the border property

Also just noticed the double semi-colons! fixed that in the next commit